internal/controller/management/projects/projects.go

```diff
@@ -856,6 +871,70 @@ func (r *reconciler) ensureDefaultUserRoles(
 	return nil
 }
 
+func (r *reconciler) ensureReceivers(
+	ctx context.Context,
+	project *kargoapi.Project,
+) error {
+	logger := logging.LoggerFromContext(ctx)
+	logger.Debug("ensuring receivers",
+		"receivers", len(project.Spec.Receivers), // nolint:staticcheck
+	)
+	var receivers []kargoapi.Receiver
+	for _, receiver := range project.Spec.Receivers { // nolint:staticcheck
```

Comment:
This needs to work based on the `ProjectConfig` for the `Project` instead of the spec (which has been deprecated).